cmd/kubeadm/app/phases/kubeconfig/kubeconfig.go

```diff
@@ -231,7 +231,7 @@ func createKubeConfigFileIfNotExists(outDir, filename string, config *clientcmda
 			return fmt.Errorf("failed to save kubeconfig file %s on disk: %v", kubeConfigFilePath, err)
 		}
 
-		fmt.Printf("[kubeconfig] Wrote KubeConfig file to disk: %q\n", filename)
+		fmt.Printf("[kubeconfig] Wrote KubeConfig file to disk: %q\n", kubeConfigFilePath)
```

Comment:
Should the verb here be `%s` for `kubeConfigFilePath`? I don't think formatting wise that this needs to be quoted/escaped.

Reply:
Yeah it's not necessarily to be quoted, but in order to be consistent with other places in kubeadm, I leave the verb as-is.

```
[controlplane] Wrote Static Pod manifest for component kube-apiserver to "/etc/kubernetes/manifests/kube-apiserver.yaml"
[controlplane] Wrote Static Pod manifest for component kube-controller-manager to "/etc/kubernetes/manifests/kube-controller-manager.yaml"
[controlplane] Wrote Static Pod manifest for component kube-scheduler to "/etc/kubernetes/manifests/kube-scheduler.yaml"
[etcd] Wrote Static Pod manifest for a local etcd instance to "/etc/kubernetes/manifests/etcd.yaml"
[init] Waiting for the kubelet to boot up the control plane as Static Pods from directory "/etc/kubernetes/manifests"
```

Reply:
I like quotes in such output. Good to visually distinguish filenames in long output.

Reply:
Makes sense to me